src/components/ContactsList/ContactRow.jsx

```diff
@@ -116,6 +116,19 @@ ContactEmail.defaultProps = {
   email: "—"
 };
 
+const ContactSelection = props => (
```

Comment:
Not sure `ContactSelection` is the best naming here... What about `SelectionCheckbox` or `ContactCheckbox`?

Reply:
Yep, I stay homegeneous with other `<ContactRow />` components with `Contact`+feature